Tests/FoundationInternationalizationTests/Formatting/DateFormatStyleTests.swift

```diff
@@ -38,6 +38,11 @@ extension AttributedString {
 @available(macOS 12.0, iOS 15.0, tvOS 15.0, watchOS 8.0, *)
 final class DateFormatStyleTests : XCTestCase {
     let referenceDate = Date(timeIntervalSinceReferenceDate: 0)
+#if FOUNDATION_FRAMEWORK
+    let expectedSeparator = "\u{202f}"
+#else
+    let expectedSeparator = " "
+#endif
```

Comment:
I very much love this solution!! Do you think we should just move it to TestSupport as a global variable?

Reply:
I don't think so. It's very data dependent so I think it's better to keep it local. It's currently used for formatting Date also.